pkg/cmd/cli/secrets/subcommand.go

```diff
@@ -0,0 +1,27 @@
+package secrets
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const SecretsRecommendedName = "secrets"
+
+func NewCmdSecrets(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	// Parent command to which all subcommands are added.
+	cmds := &cobra.Command{
+		Use:   name,
+		Short: "Manage secrets",
+		Long:  `Manage secrets`,
```

Comment:
Better long descriptions, explain secrets and what these commands can help users with